sdk/core/Azure.Core/tests/TestFramework/DiagnosticScopeValidatingInterceptor.cs

```diff
@@ -19,7 +19,14 @@ public void Intercept(IInvocation invocation)
             {
                 Type declaringType = invocation.Method.DeclaringType;
                 var ns = declaringType.Namespace;
-                var expectedName = declaringType.Name + "." + methodName.Substring(0, methodName.Length - 5);
+                var methodNameNoAysnc = methodName.Substring(0, methodName.Length - 5);
+                var expectedName = declaringType.Name + "." + methodNameNoAysnc;
+                string expectedNameNoStart = null;
+                //Fix scenario: StartCreateOrUpdateAsync -> CreateOrUpdate
+                if (methodNameNoAysnc.StartsWith("Start"))
```

Comment:
This is the generator bug, let's not workaround it here.

Reply:
great, will remove this part.